guppy/src/main/java/com/jnj/guppy/interceptor/HttpStatus.kt

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright © 2019 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package com.jnj.guppy.interceptor
+
+
+class HttpStatus {
```

Comment:
is this class needed? For instance, HttpURLConnection has most of the commonly used ones built in. https://developer.android.com/reference/java/net/HttpURLConnection.html#HTTP_BAD_REQUEST

Reply:
It's needed to map status code (int) to message. For instance 201 **Created**.

Reply:
So I don't see the need for Guppy to be responsible for defining all the Http Status/Response codes, i see that this was once part of the Apache library but deprecated in API 22: https://android.googlesource.com/platform/external/apache-http/+/refs/heads/lollipop-mr1-release/src/org/apache/http/HttpStatus.java

If you need to map status codes, use the constants available in `HttpURLConnection` but I don't see the need to have the readable strings defined in Guppy.

Reply:
Resolving as the iOS team implemented a similar method.